test/jdk/sun/security/tools/jarsigner/compatibility/Compatibility.java

```diff
@@ -23,15 +23,15 @@
 
 /*
  * @test
- * @bug 8217375 8260286 8267319
+ * @bug 8217375 8260286 8267319 8292704
```

Comment:
```suggestion
 * @bug 8217375 8260286 8267319
```
since this is test only update, you don't need to add bug id here.